lib/krane/restart_task.rb

```diff
@@ -22,6 +22,8 @@ def initialize(deployment_name, response)
     HTTP_OK_RANGE = 200..299
     ANNOTATION = "shipit.shopify.io/restart"
 
+    RESTART_TRIGGER_ANNOTATION = "kubectl.kubernetes.io/restartedAt"
```

Comment:
Is there any specific benefit to reusing the same annotation name as kubectl or is this done just to reduce annotation clutter?

Reply:
I'm fairly ambivalent about which one to use; this one cuts down noise, but can also be construed as somewhat misleading. I'm tempted to leave it as-is and change it if we get some complaints